<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOTIFICATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>UPCOMING PAYMENTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>OVERDUE ACCOUNTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>INTEREST PENALTIES</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1031</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Resources/4.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Previous">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px ;
	background-color: rgb(66, 82, 91);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(0, 255, 252);
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(0, 255, 252);
}
</string>
   </property>
   <property name="text">
    <string>Previous ◄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px ;
	background-color: rgb(0, 255, 252);
	color: rgb(0, 0, 0);
}
QPushButton:hover {
	background-color: rgb(66, 82, 91);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {
	background-color: rgb(66, 82, 91);
}
</string>
   </property>
   <property name="text">
    <string>► Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Resources/wallet.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="upcoming_events">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: rgb(10, 179, 179);
    color: white; /* Text color */
    border: 2px solid #005c5c; /* Border color */
    font-size: 12px; /* Font size */
}

QTableWidget::item {
    border: 1px solid #ffffff; /* Item border color */
}

QHeaderView::section {
    background-color: rgb(20, 130, 130); /* Header background color */
    color: white; /* Header text color */
    padding: 5px;
    font-size: 12px; /* Header font size */
}

QTableWidget::item:selected {
    background-color: rgb(0, 100, 100); /* Background color for selected items */
    color: white; /* Text color for selected items */
}
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Accounts:</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="overdue_accounts">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: rgb(10, 179, 179);
    color: white; /* Text color */
    border: 2px solid #005c5c; /* Border color */
    font-size: 12px; /* Font size */
}

QTableWidget::item {
    border: 1px solid #ffffff; /* Item border color */
}

QHeaderView::section {
    background-color: rgb(20, 130, 130); /* Header background color */
    color: white; /* Header text color */
    padding: 5px;
    font-size: 12px; /* Header font size */
}

QTableWidget::item:selected {
    background-color: rgb(0, 100, 100); /* Background color for selected items */
    color: white; /* Text color for selected items */
}
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Accounts:</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="interest_penalties">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>271</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: rgb(10, 179, 179);
    color: white; /* Text color */
    border: 2px solid #005c5c; /* Border color */
    font-size: 12px; /* Font size */
}

QTableWidget::item {
    border: 1px solid #ffffff; /* Item border color */
}

QHeaderView::section {
    background-color: rgb(20, 130, 130); /* Header background color */
    color: white; /* Header text color */
    padding: 5px;
    font-size: 12px; /* Header font size */
}

QTableWidget::item:selected {
    background-color: rgb(0, 100, 100); /* Background color for selected items */
    color: white; /* Text color for selected items */
}
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Accounts:</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>871</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>Title_2</zorder>
  <zorder>Title_3</zorder>
  <zorder>Title_4</zorder>
  <zorder>Title_5</zorder>
  <zorder>Previous</zorder>
  <zorder>Next</zorder>
  <zorder>label_2</zorder>
  <zorder>upcoming_events</zorder>
  <zorder>overdue_accounts</zorder>
  <zorder>interest_penalties</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
